web/src/lib/services/StitchService.ts

```diff
@@ -1,5 +1,5 @@
 interface StitchService {
-    callFunction(name: string, ...args: any[]): Promise<any>;
+  callFunction(name: string, ...args: any[]): Promise<any>;
```

Comment:
[nit q] Are we sure we want `args` to be variadic? If an interface above this ever exposes a variadic arg parameter to be passed as a list of args here,  it could end up being doubly nested here.

Reply:
That's true for any variadic parameter. We can change it to an array if we want, as that might be more comfortable for JS users, however, in some languages, variadic makes life a lot easier, and it's up to the dev to not make the double nesting mistake– not us.

Reply:
I think an array is the right move, this isn't going to be the interface that people use right? From what I understand our services will wrap one of these.

Reply:
Anyway, you made the change and I think that's fine

Reply:
Yeah I'd say it's the right move for JS